client/templates/Admin/Client/Search/case-search.html.twig

```diff
@@ -0,0 +1,78 @@
+{% extends '@App/Layouts/application.html.twig' %}
```

Comment:
Seeing as this is no longer client specific it would make sense to have a new directory/namespace that the templates sit under (pending the results of what we name the controller etc)